Misc/NEWS.d/next/Core and Builtins/2021-01-04-23-54-34.bpo-42819.4KO6wU.rst

```diff
@@ -0,0 +1,7 @@
+Explicitly disable bracketed paste in the interactive interpreter, even if it's
+set in the inputrc, is enabled by default (eg GNU Readline 8.1), or a user calls
+``readline.read_init_file()``. The Python REPL has not implemented bracketed
+paste support so this prevents Readline defaults or global user configuration
+from causing unexpected behavior within Python. It can still be explicitly
```

Comment:
```suggestion
``readline.read_init_file()``. The Python REPL has not implemented bracketed
paste support. Also the bracketed mode writes `"\x1b[?2004h"` escape sequence into stdout
which causes test failures in applications which don't support it. It can still be explicitly
```